src/app/wallets/get-balance-for-wallet.ts

```diff
@@ -1,25 +1,17 @@
 import { LedgerService } from "@services/ledger"
 import { updatePendingPaymentsByWalletId } from "@app/payments"
 
-import { updatePendingInvoicesByWalletId } from "./update-pending-invoices"
-
 export const getBalanceForWallet = async ({
   walletId,
   logger,
 }: {
   walletId: WalletId
   logger: Logger
 }): Promise<CurrencyBaseAmount | ApplicationError> => {
-  const [, updatePaymentsResult] = await Promise.all([
-    updatePendingInvoicesByWalletId({
-      walletId,
-      logger,
-    }),
-    updatePendingPaymentsByWalletId({
-      walletId,
-      logger,
-    }),
-  ])
+  const updatePaymentsResult = await updatePendingPaymentsByWalletId({
```

Comment:
note: I think we could move this in a hourly cronjob. Eventually this fallback method is really only used when trigger miss an event. this would be more one step to make the getBalanceForWallet totally independant of lnd

Reply:
note for future PR